Entities/Attachable.cpp

```diff
@@ -87,6 +89,7 @@ int Attachable::Create(const Attachable &reference)
     m_DamageCount = reference.m_DamageCount;
     m_OnlyLinForces = reference.m_OnlyLinForces;
 	m_InheritsRotAngle = reference.m_InheritsRotAngle;
+	m_CanCollideWithTerrain = reference.m_CanCollideWithTerrain;
```

Comment:
I'm making `m_CollidesWithTerrain` also copy over in clone Create. If I'm mistaken in this, please let me know.

Reply:
Doing this makes collisions not enable even if all conditions are set for them to.